index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <title>Document</title>
+
+  <link rel="stylesheet" href="style.css">
+  <!-- Frontawesome CDN -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" />
+</head>
+<body>
+  <div class="ellipse-one"></div>
+  <div class="ellipse-two"></div>
+  <header class="header">
+    <div class="logo">
+      <a href="#" class="link" target=" blank">DU</a>
+    </div>
+    
+    <button type="button" class="hamburger">
+      <span class="bars"></span>
+    </button>
+  </header>
+  <div class="containner">
+    <main class="main">
+      <section class="container border-bottom">
+        <h1 class="item1">Dan Ushindi</h1>
+        <h2 class="item2">Software Developer</h2>
+        <div class="literature">
+          <p class="item3">I can help you build a product ,
+             feature or website Look through  
+             some of my work and experience! If 
+             you like what you see and have a 
+             project you need coded,  don’t 
+             hestiate to contact me.</p>
+        </div>
+      </section>
+      <section class="tools border-bottom">
+        <div class="language">
+          <p class="tittle">Language</p>
+          <p>JavaScript</p>
+          <p>Ruby</p>
+          <p>Html</p>
+          <p>Css</p>
+        </div>
+        <div class="frameworks">
+          <p class="tittle">frameworks</p>
+          <p>Bootstrap</p>
+          <p>Ruby on Rails</p>
+          <p>RSPec</p>
+          <p>Capy Bary</p>
+          <p>Selenium</p>
+        </div>
+        <div class="skills">
+          <p class="tittle">skills</p>
+          <p>Codekit</p>
+          <p>Github</p>
+          <p>Codpen</p>
+          <p>Gitlab</p>
+          <a href="#"><p>Terminal</p></a>
+        </div>
+      </section>
+      <section class="footer">
+        <ul class="icon">
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-linkedin-in"></i></li>
+          <li><i class="fab fa-angellist"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
```

Comment:
- [Required] :
Please make sure that social media icons ("LinkedIn", "Angellist") are in a `ul` tag, each of them in a `li `tag, each icon has a link (`a` tag) because you need to need to add your social media links to them ^^